cypress/jenkins/init.sh

```diff
@@ -144,6 +144,15 @@ create_initial_clusters() {
       RANCHER_VERSION=$(helm search repo "rancher-${RANCHER_HELM_REPO}" --devel --versions | grep "${version_string}" | head -n 1 | cut -f2 | tr -d '[:space:]')
     fi
     corral config vars set rancher_image_tag "${RANCHER_IMAGE_TAG}"
+    
+    # Save all values to a file for the Slack notification script
+    cat > "${WORKSPACE}/notification_values.txt" << EOF
```

Comment:
there's a case in line 243 that modifies the CYPRESS_TAGS to append @noVai for certain Rancher versions. The `notification_values.txt` gets the initial CYPRESS_TAGS value, missing the @noVai modification. Recommendation: So I recommend moving the `cat > "${WORKSPACE}/notification_values.txt"` block to the end of or after this case that modifies the CYPRESS_TAGS to @noVai.